packages/terra-search-field/src/SearchField.jsx

```diff
@@ -284,34 +297,44 @@ class SearchField extends React.Component {
       : undefined;
 
     return (
-      <div {...customProps} className={searchFieldClassNames}>
-        <div className={cx('input-group')}>
-          <input
-            {...additionalInputAttributes}
-            className={inputClass}
-            type="search"
-            placeholder={placeholder}
-            onChange={this.handleTextChange}
-            disabled={isDisabled}
-            aria-label={inputText}
-            aria-disabled={isDisabled}
-            onKeyDown={this.handleKeyDown}
-            onInput={this.handleInput}
-            ref={this.setInputRef}
-          />
-          {clearButton}
+      <div className={cx('search-container')}>
+        {isLabelVisible
+        && (
+        <label className={cx('label')}>{groupName}</label>
+        )}
+        <div className={cx('search-container')} role="search" aria-label="Search">
+          <div className={cx('search-container')} role="group" aria-label={groupName}>
+            <div {...customProps} className={searchFieldClassNames}>
+              <div className={cx('input-group')}>
+                <input
+                  {...additionalInputAttributes}
+                  className={inputClass}
+                  type="search"
+                  placeholder={placeholder}
+                  onChange={this.handleTextChange}
+                  disabled={isDisabled}
+                  aria-label={inputText}
```

Comment:
Is this correct for accessibility?

Reply:
The `inputText` variable was named poorly. It it updated now to `inputAriaLabelText` which is a custom aria-label passed in by consumer.